src/main/java/by/exadel/internship/entity/user/TechUser.java

```diff
@@ -0,0 +1,14 @@
+package by.exadel.internship.entity.user;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "techUser")
```

Comment:
Will you store tech user in the other table?

Reply:
done